utils/filesystem/files.go

```diff
@@ -1165,6 +1159,34 @@ func (fs *VFS) copyZipFile(dest string, zippedFile *zip.File) (err error) {
 	return
 }
 
+func determineUnzippedFilepath(dest string) (string, error) {
+	destinationPath := dest
```

Comment:
```suggestion
func determineUnzippedFilepath(destinationPath string) (string, error) {
```

`dest` doesn't seem to be used so may as well change the name of the argument.